core/client/app/controllers/setup/two.js

```diff
@@ -48,6 +48,10 @@ export default Ember.Controller.extend(ValidationEngine, {
         });
     },
 
+    onEmailTextChange: function () {
+        Ember.run.debounce(this, 'send', 'preValidate', 'email', 700);
+    }.observes('email'),
```

Comment:
Just a style note but we've switched away from using prototype extensions like these now, if you could use `Ember.observer('email', function () {})` instead that would be :+1: